src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/HttpGoogleCloudStorageDAOSpec.scala

```diff
@@ -59,8 +65,8 @@ class HttpGoogleCloudStorageDAOSpec extends FlatSpec with Matchers with Integrat
     val ownerResource = directory.groups.get(ownerGroup).execute()
 
     // check that the creator is an owner, and that getACL is consistent
-    gcsDAO.getMaximumAccessLevel(testCreator.userEmail, testWorkspaceId) should be (WorkspaceAccessLevel.Owner)
-    gcsDAO.getACL(testWorkspaceId).acl should be (Map(testCreator -> WorkspaceAccessLevel.Owner))
+    Await.result(gcsDAO.getMaximumAccessLevel(testCreator.userEmail, testWorkspaceId), Duration.Inf) should be (WorkspaceAccessLevel.Owner)
```

Comment:
why are we now awaiting these when we didnt before, and why are we not using futures